src/towncrier/test/test_settings.py

```diff
@@ -192,6 +194,22 @@ def test_towncrier_toml_preferred(self):
         config = load_config(project_dir)
         self.assertEqual(config.package, "a")
 
+    def test_load_no_config(self):
+        """
+        Calling the root CLI without an existing configuration file in the base directory,
+        will exit with ConfigError.
```

Comment:
I think that for this test, the ConfigError  is not really important.

this looks to me like an end-to-end test.
So the important part is that the error message is sent to stdout (and not stderr) and process will exit with code 1 (non-zero code)